python-package/lightgbm/sklearn.py

```diff
@@ -735,6 +740,7 @@ def fit(self, X, y,
                                        early_stopping_rounds=early_stopping_rounds,
                                        verbose=verbose, feature_name=feature_name,
                                        categorical_feature=categorical_feature,
+                                       init_model=init_model,
```

Comment:
Move according to the new order. Also, please place it at the same line as `callbacks`.